src/dashboardgeneral.html

```diff
@@ -44,6 +44,26 @@ <h2>${Advanced}</h2>
                         </div>
                         <div class="fieldDescription">${LabelCachePathHelp}</div>
                     </div>
+
+                    <div class="inputContainer">
+                        <div style="display: flex; align-items: center;">
+                            <div style="flex-grow:1;">
+                                <input is="emby-input" id="txtMetadataPath" label="${LabelMetadataPath}" autocomplete="off" />
```

Comment:
why `autocomplete=off`?

Reply:
I was just leaving that in from the other examples, didn't see a reason to change it specifically. Do you think it should be enabled?

Reply:
I actually have no preference neither for `on` nor `off` TBH :) So was just curious...